src/ReactNativeZoomableView.tsx

```diff
@@ -279,6 +318,8 @@ class ReactNativeZoomableView extends Component<
 
     this.props.onTransform?.(zoomableViewEvent);
 
+    this.props.onStaticPinPositionMove?.(this._staticPinPosition());
```

Comment:
every time `onStaticPinPositionMove` is called, we should follow by a debounced `onStaticPinPositionChange`